test/workers/suspend_inactive_accounts_worker_test.rb

```diff
@@ -26,7 +26,7 @@ class SuspendInactiveAccountsWorkerTest < ActiveSupport::TestCase
 
   test 'ignores accounts that fail to validate' do
     valid_tenant = FactoryBot.create(:simple_provider)
-    invalid_tenant = FactoryBot.build(:simple_provider, org_name: '@@@')
+    invalid_tenant = FactoryBot.build(:simple_provider, org_name: '')
```

Comment:
because after https://github.com/3scale/porta/pull/4118 `org_name: '@@@'` will not fail validation on update.

Reply:
honestly, this is also weird - failing to suspend an account with invalid name, more likely it should manage to suspend it. And ignore validations for the purposes of suspending or making as deleted. But whatever, I wrote to the other PR.

Reply:
To me this should be done by stubbing `validate` instead of relying in underlying logic that is clearly unreliable.